docs/docs-site/content/developer/development/_index.md

```diff
@@ -226,7 +226,7 @@ Once we get ship it from at least one reviewer, we can push the changes to maste
 * Add the commit link to the ticket and mark it as resolved
 
 
-### Sump-up
+### Sump
```

Comment:
Actually --> Sum-up ?

Reply:
That looks likes a better heading, will change it.